### ISSUE TITLE
Mailto links don't open Thunderbird if it's the default mail client (Linux)

### ISSUE THREAD

**Issue by Lombress** [QupZilla/qupzilla#2362]
If i set Thunderbird as the default mail client in Opensuse 42.2 with KDE (don't know about other desktops or distributions), mailto links will still be opened in Kmail by Qupzilla (but not by Chrome or Libreoffice)

**Comment by Martii** [comment]
Test link: user@example.com

**Comment by iondissonance** [comment]
openSUSE Tumbleweed, KDE 4.10.1, Thunderbird 52.1.1, can't reproduce.

**Comment by Martii** [comment]
d6b142bba2d9c73aff1201cacefaffc0a7423376
ROSA

Can't reproduce either.

**Comment by iondissonance** [comment]
openSUSE Leap 42.2, **reproduce**.

**Comment by Martii** [comment]
Are either one of you using Moz's distribution or is it a distro package? I'm still updating my VM of 42.2.

**Comment by iondissonance** [comment]
All works, if you install QupZilla 2.0.2 from [KDE:Extra repository](https://www.qupzilla.com/download#suse).

**Comment by Martii** [comment]
KDE System Settings &rarr; Default Applications &rarr; Email Client settings?

**Comment by iondissonance** [comment]
Yep.

**Comment by Martii** [comment]
Qt will take a while for me to update so I can build QupZilla on the VM... I can't keep up with updates in my VMs until an issue arises. ;)

**Comment by Martii** [comment]
OpenSUSE 42.2 *(Leap VM)*
Qt 5.9 release
d6b142bba2d9c73aff1201cacefaffc0a7423376
KDE System Settings → Default Applications → Email Client settings set first to `thunderbird %u`

Unable to reproduce e.g. it's working for me... KDE Launch Feedback shows Tbirds icon and Help &rarr; About is showing Thunderbird 52.1.1 *(64 bit)* from distro package.

**Comment by Lombress** [comment]
Still doesn't work to me... Qt version is 5.6.1, Thunderbird is 52.1.1, Qupzilla is 2.0.2 from Kde Extra repo

**Comment by iondissonance** [comment]
Try this:

`xdg-mime default thunderbird.desktop x-scheme-handler/mailto`

**Comment by iondissonance** [comment]
Check this option in Thunderbird settings 
![image](https://user-images.githubusercontent.com/18717061/27255657-5e05c472-53ab-11e7-933b-b85f114b6205.png)

**Comment by Lombress** [comment]
I've tried both, it still opens Kmail. Do you think I can try uninstalling Kmail without damaging KDE?

**Comment by iondissonance** [comment]
Edit /etc/zypp/zypper.conf, uncomment installRecommends and set to no.
Then
`sudo zypper remove kmail`

**Comment by Lombress** [comment]
done, but it tries to open Kmail. An application called "kmailservice" seems to start, then nothing happens

**Comment by Lombress** [comment]
Finally solved! Thanks to what I found here [https://wiki.archlinux.org/index.php/default_applications](url) I looked for files to edit and I found the right one.

After setting Thunderbird as default mail client in KDE System Settings and giving the command  `xdg-mime default thunderbird.desktop x-scheme-handler/mailto` from the terminal, I also edited the file /usr/share/applications/mimeinfo.cache using "File Manager - Super User Mode" and near _x-scheme-handler/mailto=_ I deleted _org.kde.kmail.desktop;kde4-kmailservice.desktop_ and wrote _thunderbird.desktop_. Now it works.

Do I have to click "close issue" right now?
